### PR TITLE
scott's suggestions/revisions

### PR DESCRIPTION
**Pull request by sdc50**
@acatlett8 Here are my edits/suggestions to help clarify wording. Let me know if there is anything else you need me to add.

### CODE REVIEW

File: papers/Catlett_Amanda/UCET.rst [hunk 1]
```diff
@@ -206,8 +206,7 @@ would be, if a user chooses input values that make it so the application does no
 will appear under the output header and replace all output values. For a more concrete example: Linear Wave Theory
```
Comment (sdc50) [inline papers/Catlett_Amanda/UCET.rst:206]: @acatlett8  A figure that shows the warning message would be really helpful.

File: papers/Catlett_Amanda/UCET.rst [hunk 2]
```diff
@@ -162,11 +161,11 @@ UCET utilizes the following widgets to gather user input:
 
 UCET utilizes indicators.Number, Tabulator, and graphs to visualize the outputs of the coastal engineering models.  A
 single number is shown using indicators.Number and graph data is displayed using the Tabulator widget to show the data
-of the graph. The graphs are created using HoloViews and will have tool options such as pan, zooming, and saving.
+of the graph. The graphs are created using HoloViews and have tool options such as pan, zooming, and saving.
 Buttons are used to calculate, save the current run, and save the graph data.
 
 All of these widgets are organized into 5 panels: title, options, inputs, outputs, and graph. The
-BaseGui/WaterTypeGui/TabularDataGui have methods that will organize the widgets within the 5 panels that most tools
+BaseGui/WaterTypeGui/TabularDataGui have methods that organize the widgets within the 5 panels that most tools
```
Comment (sdc50) [inline papers/Catlett_Amanda/UCET.rst:168]: @acatlett8 Another figure here that illustrates the 5 panels would help clarify this part.

File: papers/Catlett_Amanda/UCET.rst (hunk 1 above)
Comment (acatlett8) [reply papers/Catlett_Amanda/UCET.rst:206]: That is a good idea! I will add!

File: papers/Catlett_Amanda/UCET.rst (hunk 2 above)
Comment (acatlett8) [reply papers/Catlett_Amanda/UCET.rst:168]: Good thing I made this a while back! I will add it!!

**Review by acatlett8 — APPROVED** [review]

**Pull request merged**